Controllers/UserController.cs

```diff
@@ -28,5 +40,58 @@ public async Task<ActionResult<UserResponseDTO>> UpdateUser([FromForm] UserUpdat
 
             return Ok(_mapper.Map<UserResponseDTO>(result));
         }
+
+        [HttpGet("posts")]
+        public async Task<ActionResult<IEnumerable<UserPostResponseDTO>>> GetUserPosts(
+            [FromQuery] PaginationRequestDTO request
+        )
+        {
+            var result = await _postService.GetUserPosts(request, GetUserAddress());
+
+            return Ok(_mapper.Map<IEnumerable<UserPostResponseDTO>>(result));
+        }
+
+        [HttpGet("chat-profile")]
+        public async Task<ActionResult<UserChatProfileResponseDTO>> GetUserChatProfile()
+        {
+            var result = await _chatService.GetUserChatProfile(GetUserAddress());
+
+            return Ok(_mapper.Map<UserChatProfileResponseDTO>(result));
+        }
+
+        [HttpGet("followers")]
+        public async Task<ActionResult<IEnumerable<FollowerResponseDTO>>> GetUserFollowers(
+            [FromQuery] PaginationRequestDTO request)
+        {
+            var result = await _followerService.GetUserFollowers(GetUserAddress(), request);
+            var dto = result.Select(x => new FollowerResponseDTO { UserAddress = x.UserAddress, FollowingSince = x.CreatedAt });
+
+            return Ok(dto);
+        }
+
+        [HttpGet("followings")]
```

Comment:
prefix URL with userAddress and use BindUser annotation, we also need endpoints for getting the followings of other users, not just the currently logged in user